google-http-client/src/main/java/com/google/api/client/http/GenericUrl.java

```diff
@@ -503,6 +561,20 @@ public void appendRawPath(String encodedPath) {
    *     or {@code ""} input
    */
   public static List<String> toPathParts(String encodedPath) {
+    return toPathParts(encodedPath, false);
+  }
+
+  /**
+   * Returns the decoded path parts for the given encoded path.
```

Comment:
now always decoded, right?

Reply:
Done

Reply:
I don't see any change. I suggest

`Returns the path parts (decoded if not {@code verbatim}) ...`